docs/src/pages/guide/how-to/00-llm-models.mdx

```diff
@@ -35,6 +35,8 @@ async function main() {
 main();
 ```
 
+The code for this example can be found [here](https://github.com/mastra-ai/mastra/tree/main/examples/llm).
```

Comment:
After the PR is merged. I don't think it will link to 404 again.
The example is part of this PR.
<img width="269" alt="image" src="https://github.com/user-attachments/assets/3dae761e-63d7-4496-978b-429b48fa2154">